### DISCUSSION TITLE
required field for pictures

### DISCUSSION THREAD

**Discussion by joaopaulozz** [RensTillmann/super-forms#54] [Q&A] [answered]
Hi, is it possible to leave the pdf / image field as a mandatory field?

**Comment by RensTillmann** [comment, marked as answer]
Yes: https://webrehab.zendesk.com/hc/en-gb/articles/360016993478-How-to-make-the-file-upload-required-mandatory-

**Reply by joaopaulozz** [reply]
Great!!!
